connectors/byoc.py

```diff
@@ -630,26 +566,38 @@ async def _sync_done(self, job, status, result, exception=None):
             await job.fail(exception, ingestion_stats=ingestion_stats)
         elif status == JobStatus.SUSPENDED:
             await job.suspend(ingestion_stats=ingestion_stats)
-            logger.info(f"Sync suspended, Job id: {job.id}")
         else:
             await job.done(ingestion_stats=ingestion_stats)
 
         job = await job.reload()
         await self.sync_done(job)
         logger.info(
             f"[{job_id}] Sync done: {doc_updated + doc_created} indexed, {doc_deleted} "
-            f" deleted. ({int(time.time() - self._start_time)} seconds)"
+            f" deleted. ({int(time.time() - start_time)} seconds)"
```

Comment:
```suggestion
            f" deleted. (Took {int(time.time() - start_time)} seconds)"
```

Maybe `{int(time.time() - start_time)}` could also be a variable called `sync_duration` or something similar

Reply:
There are many tests (mainly in `test_sync.py`), which asserts some log presence, e.g:
https://github.com/elastic/connectors-python/blob/e75e7b30f5124596151451d296fca7022bb728af/connectors/tests/test_sync.py#L425

I'm not in favor of this style, but I don't intend to change them in this PR.

The proposed change can be done when I refactor `SyncService`.

Reply:
Makes sense, would love to see removing some of the logging assertions 👍

Reply:
It's not very hard to add as a test fixture (`synced_data` for example) with a few attributes and assertions methods, since the data is present in the classes.